l2geth/core/evm.go

```diff
@@ -65,10 +65,10 @@ func NewEVMContext(msg Message, header *types.Header, chain ChainContext, author
 			Transfer:        Transfer,
 			GetHash:         GetHashFn(header, chain),
 			Origin:          msg.From(),
-			Coinbase:        beneficiary,
+			Coinbase:        common.HexToAddress("0x4200000000000000000000000000000000000011"), // Coinbase is the fee vault.
```

Comment:
This should be defined in a single place so it doesn't need to have the string parsed each time

Reply:
Any other notes? Or should be good for approval once this is fixed?

Reply:
If you think its possible to add some unit tests in geth for this then that would be the most ideal, but I'd approve it if you think its going to be too big of a pain.